examples/nodejs-nestjs/src/app.module.ts

```diff
@@ -0,0 +1,19 @@
+import { Module } from '@nestjs/common';
+import { ConfigModule } from '@nestjs/config';
+
+import { AppController } from './app.controller.js';
+import { AppService } from './app.service.js';
+
+import { ProtectedModule } from './protected/protected.module.js';
+
+import { ArcjetGuard } from './arcjet/arcjet.guard.js';
+
+@Module({
+  imports: [ConfigModule.forRoot(), ProtectedModule],
+  controllers: [AppController],
+  providers: [
```

Comment:
Add a comment here about what this does, why Arcjet is integrated here, and a link to the NestJS docs related to this?